lldb/include/lldb/Core/Telemetry.h

```diff
@@ -56,13 +60,83 @@ struct LLDBBaseTelemetryInfo : public llvm::telemetry::TelemetryInfo {
   void serialize(llvm::telemetry::Serializer &serializer) const override;
 };
 
+/// Describes the exit status of a debugger.
+struct ExitDescription {
+  int exit_code;
+  std::string description;
+};
+
+struct DebuggerTelemetryInfo : public LLDBBaseTelemetryInfo {
+  std::string username;
+  std::string lldb_git_sha;
```

Comment:
You may not have a git hash. How about storing the lldb version instead?

Reply:
Right - that's the general idea. But for our specific use-case, (as far as I remember) we need the git-sha to figure out which release it was (the info will be supplied at build time). I don't remember the exact detail, though.

will make this a generic "lldb_version" for now.